pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumloadbalancerippools.yaml

```diff
@@ -58,20 +58,47 @@ spec:
             description: Spec is a human readable description for a BGP load balancer
               ip pool.
             properties:
+              allowFirstLastIPs:
```

Comment:
Would there ever be a need to have this be per block?

Reply:
I don't think so. You can always make 2 pools one with and one without this property set and put CIDRs in the corresponding pools